### ISSUE TITLE
Can't focus currency selector on expenses page

### ISSUE THREAD

**Issue by lornajane** [ibm-watson-data-lab/advocated#2]
This field seems to be missing from the tab index and I can't "get" it with my accessibility tools.  If I click, it's fine but it would be great if this could just be a standard dropdown or something for speedier data entry

**Comment by glynnbird** [comment]
This is because the vue-material framework helpfully uses non-standard controls to make them look like "Material Design" controls. I'll force them to be good old-fashioned `select` HTML controls, which will be keyboard accessible.
